webui/react/src/pages/TrialDetails/TrialDetailsLogs.tsx

```diff
@@ -92,6 +93,9 @@ const TrialDetailsLogs: React.FC<Props> = ({ experiment, trial }: Props) => {
 
   useEffect(() => {
     settings.searchText?.length && setSearchOn(true);
+    setSearchInput((prev) =>
+      prev === (settings.searchText || '') ? prev : settings.searchText || '',
```

Comment:
super nit: I think this could be slightly simplified to `(prev === settings.searchText ? prev : settings.searchText) ?? ''`